Apparently declaring this external variable "weak" (and seemingly for no 
reason) causes a segfault.  Go figure.

--- origsrc/gxine-0.5.906/src/utils.c	2010-12-05 05:38:44.000000000 -0600
+++ src/gxine-0.5.906/src/utils.c	2011-12-11 15:37:18.075465200 -0600
@@ -797,8 +797,10 @@ void config_update_default (const char *
 
 /* List a subset of video plugins */
 
+#if 0
 extern const char *const *xine_list_video_output_plugins_typed (xine_t *, uint64_t)
 	__attribute__((__weak__));
+#endif
 
 const char *const *gxine_list_video_output_plugins (xine_t *xine)
 {
